lib/ui/widgets/appSelectorView/not_installed_app_item.dart

```diff
@@ -54,12 +55,28 @@ class _NotInstalledAppItem extends State<NotInstalledAppItem> {
                     ),
                   ),
                   const SizedBox(height: 4),
-                  const Text('App not installed.'),
-                  const SizedBox(height: 4),
+                  I18nText(
+                    'appSelectorCard.notInstalled',
+                    child: Text(
+                      '',
+                      style: TextStyle(
+                        color:
+                        Theme.of(context).textTheme.titleLarge!.color,
+                      ),
+                    ),
+                  ),
                   Row(
                     children: [
-                      Text(
-                        'Suggested: ${widget.suggestedVersion.isEmpty ? 'All versions' : 'v${widget.suggestedVersion}'}',
+                      I18nText(
+                        FlutterI18n.translate(
+                          context,
+                          'suggested',
+                          translationParams: {
+                            'version': widget.suggestedVersion.isEmpty
+                                ? 'All versions'
```

Comment:
Did you happen to miss using the key here instead?

Reply:
Which key is missing?

Reply:
You could create an i18n key for the string "All versions" here

Reply:
Can you check the new commits?